_i18n/nl.yml

```diff
@@ -2,6 +2,10 @@ langs:
   en: English
   nl: Nederlands
 
+site_pages:
+  about: Over TWC NL
```

Comment:
I was thinking "Over" sounds a bit strange in Dutch, so that's why I added that! But maybe it's a standard term on websites now. :) Feel free to edit and commit to the branch if you'd prefer it out (I don't have laptop atm)!